elixir/Dockerfile

```diff
@@ -29,25 +29,26 @@ COPY config/ config/
 
 ARG MIX_ENV="prod"
 RUN mix deps.get --only ${MIX_ENV}
-RUN mix deps.compile --skip-umbrella-children
+RUN --mount=type=cache,target=./_build mix deps.compile --skip-umbrella-children
 
 COPY priv priv
 COPY apps apps
 
 ARG APPLICATION_VERSION=0.0.0-dev.docker
 
 # Install pipeline and compile assets for Web app
-RUN cd apps/web \
+RUN --mount=type=cache,target=./_build cd apps/web \
     && mix assets.setup \
     && mix assets.deploy
 
 # Compile the release
-RUN mix compile
+RUN --mount=type=cache,target=./_build mix compile
 
 COPY rel rel
 
 ARG APPLICATION_NAME
-RUN mix release ${APPLICATION_NAME}
+RUN --mount=type=cache,target=./_build mix release ${APPLICATION_NAME}
+RUN --mount=type=cache,target=./_build cp -r /app/_build/${MIX_ENV}/rel/${APPLICATION_NAME} /app/${APPLICATION_NAME}
```

Comment:
This bit is important, we need to `cp` the final artifacts out of the cache, otherwise the next container can't access them.